core/src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -461,12 +463,48 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
     }
   }
 
-  it should "fail to get usage when there are no storage logs" in {
-    intercept[GoogleStorageLogException] {
-      Await.result(gcsDAO.getBucketUsage(RawlsBillingProjectName(testProject), "no-bucket-here-" + UUID.randomUUID.toString), Duration.Inf)
+  it should "return 0 usage when there are no storage logs and the bucket is empty" in {
+    val storage = gcsDAO.getStorage(gcsDAO.getBucketServiceAccountCredential)
+
+    val projectName = RawlsBillingProjectName(testProject)
+    val logBucketName = gcsDAO.getStorageLogsBucketName(projectName)
+    val bucketName = "services-dao-spec-" + UUID.randomUUID.toString
+    val bucket = new Bucket().setName(bucketName)
+    val bucketInserter = storage.buckets.insert(testProject, bucket)
+    Await.result(retry(when500)(() => Future { bucketInserter.execute() }), Duration.Inf)
+
+    try {
+      val usage = Await.result(gcsDAO.getBucketUsage(projectName, bucketName), Duration.Inf)
+      usage should be (0)
+    } finally {
+      Await.result(gcsDAO.deleteBucket(bucketName, bucketDeletionMonitor), Duration.Inf)
     }
   }
 
+  it should "return 'Not Available' when there are no storage logs and the bucket is NOT empty" in {
```

Comment:
Good point. I had written it in the JIRA ticket, but I agree that it's worth repeating here.